spec/features/people_spec.rb

```diff
@@ -10,9 +10,24 @@
 
   end
 
+  scenario 'try to add a person with a duplicate identification' do
+    person = build(:person, identification: '10373737')
+
+    2.times do
+      expect(create_person(person))
+    end
+
+    expect(page).to have_content 'Identification has already been taken'
+  end
+
   scenario 'edits a person' do
-    create(:person)
-    update_person
+    person = create(:person)
+    visit people_path
+    click_link("Edit", match: :first)
+
+    fill_fields(person)
```

Comment:
:smile: I think you're right, I have changed it here [86e5220](https://github.com/andru1989/population/commit/86e5220aff6011c3f2799b1182a7fd7c9d1c3051)